### PR TITLE
Notice for plugins

### PR DESCRIPTION
**Pull request by ppot**
Include notice for plugins affecting the BrowserWindow UI will take effect on new created UI Window. Not the current one.

### CODE REVIEW

File: website/index.html [hunk 1]
```diff
@@ -1217,6 +1221,9 @@ <h3 id="extensions-module-loading"><a href="#extensions-module-loading">Module l
         <p>The user can hot-load and hot-reload plugins by pressing
         Command + R (refresh). Please strive to make plugins that don't
         require a complete restart of the application to work.</p>
+        
+        <h4 >Notice</h4>
```
Comment (albinekb) [inline website/index.html]: space inside the <h4>

File: website/index.html [hunk 2]
```diff
@@ -1217,6 +1221,9 @@ <h3 id="extensions-module-loading"><a href="#extensions-module-loading">Module l
...
+        <p> Plugins affecting the `BrowserWindow` will the effect on new windows after hot-reload.</p>
```
Comment (albinekb) [inline website/index.html]: Space after <p>

**Comment by ppot** [comment]
@albinekb It's called vim 😂 quick edit files

**Comment by albinekb** [comment]
re-open to trigger stalled tests

**Comment by chabou** [comment]
This PR added a `.DS_Store` file

**Pull request merged**